logging/src/main/resources/schema/jboss-as-logging_5_0.xsd

```diff
@@ -575,7 +584,8 @@
             <xs:element name="meta-data" type="propertiesType" minOccurs="0">
                 <xs:annotation>
                     <xs:documentation>
-                        Sets the meta data to use in the structured format. Properties will be added to each log message.
+                        Sets the meta data to use in the structured format. Properties will be added to each log
```

Comment:
is it a typo? The only change is an unnecessary new line

Reply:
Yeah. It probably did got formatted to in my IDE. I'll remove that change.